react/ui-components/src/atoms/stories/StaticSlideOverMenu.stories.js

```
@@ -0,0 +1,1024 @@
+  defaultClosedWidth: "",
+  defaultOpenWidth: "",
+  addClose:false,
+  closedContents:[],
+  closedSections:[],
+  closedHeader:[],
+  closedFooter:[],
+  hideArrow:false
+};
+
+const children = [
+  <div>This is static content on the right!</div>,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)